pkg/validator/server.go

```diff
@@ -35,6 +35,7 @@ var validationHandlers = map[string]admitFunc{
 	"KubeDirectorApp":     admitAppCR,
 	"KubeDirectorCluster": admitClusterCR,
 	"KubeDirectorConfig":  admitKDConfigCR,
+	"ConfigMap":           admitConfigMap,
```

Comment:
For this handler to be called, don't we also need a change in createAdmissionService (in validator/util.go) to register a webhook for it?

Hmm, I guess the current handler is a NOP so it doesn't matter. What were you considering for putting in that handler?

If we don't need a validator for ConfigMap then we should remove that code, and also remove it from the list of validationHandlers here.

Reply:
Yes, I was doing some stuff there. Will remove now